definitions/divorce/json/CaseTypeTab.json

```diff
@@ -1753,7 +1753,6 @@
     "TabLabel": "Payment",
     "TabDisplayOrder": 7,
     "CaseFieldID": "D8HelpWithFeesReferenceNumber",
-    "UserRole": "caseworker-divorce-courtadmin",
```

Comment:
It was done automatically? Shall I revert it?

Reply:
Yes please revert all these changes, probably your base excel was not the latest one

Reply:
reverted